src/rpc/coinjoin.cpp

```diff
@@ -440,6 +440,16 @@ static RPCHelpMan getcoinjoininfo()
                                                 {RPCResult::Type::STR, "address", ""},
                                             }
                                         },
+                                        {RPCResult::Type::ARR, "platform_p2p", /*optional=*/true, "Addresses used for Platform P2P",
```

Comment:
IMO coinjoin related RPC `getcoinjoininfo` should not return `platform_https` and `platform_p2p`;  consider just removing these fields comletely.
Probably you need to add helper "get p2p addreses only" for NetInfo, something like that:
```
-        obj.pushKV("addresses", mixingMasternode->pdmnState->netInfo->ToJson());
+        obj.pushKV("addresses", mixingMasternode->pdmnState->netInfo->FilterBy(NetInfoPurpose::P2P_CORE)->ToJson());
```

Reply:
CoinJoin-specific RPCs will now return `addrs_core_p2p` instead of `addresses`, resolved in latest push.